transforms/paths/responses.js

```diff
@@ -11,13 +11,25 @@ const formatResponses = (values, examples) => values.reduce((acc, value) => {
     errorMessage(`Status ${status} is not valid to create a response`);
     return {};
   }
+
+  const exampleList = examples[status];
+  if (exampleList && contentType === 'application/json') {
+    Object.keys(exampleList)
+      .filter(k => typeof exampleList[k].value === 'string')
+      .forEach(k => {
+        try {
+          exampleList[k].value = JSON.parse(exampleList[k].value);
+        } catch (ignored) { }
+      });
```

Comment:
I think it looks great! we are going to merge this but we will remove this empty catch as well as the linter rule in a future PR to add this error message https://github.com/BRIKEV/express-jsdoc-swagger/blob/acbaec49e50e8442c6cd28e87c8f1285af0560f4/transforms/utils/errorMessage.js

This will notify the user that the example was not written well 👍 if you could do that @thuydx55 it would be great. If you cannot do it, don't worry because we will merge this at the end of the day to do a new release.